README.md

```diff
@@ -43,6 +43,12 @@ PDF.js is built into version 19+ of Firefox, however the extension is still avai
 Chrome, go to `Tools > Extension` and load the (unpackaged) extension from the
 directory `build/chromium`.
 
+## Using Core Library in a Web Application
+
+The package may be installed via npm:
```

Comment:
We have more information provided on the subject at https://github.com/mozilla/pdf.js/wiki/Frequently-Asked-Questions#gh-pages . You can use the last paragraph from there almost as is. (I wish there will be a simple use case listed too)

Move this section below "Building PDF.js" to not confused contributors to the project.